packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
-## v3.0.0-dev
+## v1.0.0 (June 15, 2023)
 
-- Upgrade React v18, React DOM v18, React-hook-form v7, and Chakra 2 libraries [#1166](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1166)
-- Add app `above-header` and product-list `above-page-header` convenience components [#1183](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1183)
-- Remove `cross-fetch` dependency [#1160](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1160)
+-   First public release of package on npm under `@salesforce/retail-react-app`
+-   Upgrade React v18, React DOM v18, React-hook-form v7, and Chakra 2 libraries [#1166](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1166)
+-   Add app `above-header` and product-list `above-page-header` convenience components [#1183](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1183)
+-   Remove `cross-fetch` dependency [#1160](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1160)
```

Comment:
let's remove this line, since we added `cross-fetch` in the translation PR https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1261/files#diff-31a530759d3695d89427d1122bfd39ae7a234f7717115083624c8ed0a8f87cc0R11